src/site/content/en/lighthouse-performance/third-party-summary/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Reduce the impact of third-party code
```

Comment:
As we've discussed in other threads, we want the title to match the `failureTitle` of each audit. This audit was originally under **Diagnostics** so it only had 1 title (`Third-party usage`) and was informational only. It was recently moved to **Opportunities** and become a pass/fail audit. The `failureTitle` is now `Reduce the impact of third-party code` so that's what we want to match. 

https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/third-party-summary.js#L20